docker-compose.yml

```diff
@@ -59,16 +73,24 @@ services:
       POSTGRES_DB: tasks
     ports:
       - 5433:5432
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "tad"]
+      interval: 10s
+      timeout: 5s
+      retries: 3
 
   # Core Services
   analysismanager:
-    image: well5a/analysis-manager:latest
+    image: well5a/analysis-manager:testing
```

Comment:
should we really use the testing images here in the main branch?

Reply:
sadly, yeah, as the dev branches of our repos (tagged with testing here) are way ahead of the main branch and currently are the only ones including `curl`

Reply:
- hmm okay maybe we should merge dev in main then in all other branches :D but well - don't take it as a blocker here

Reply:
yeah, I'd much rather prefer using the latest (i.e., main branch) tag but like I mentioned, the required `curl` command for the healthy checks are currently only available in the testing images

Reply:
@eliasmueller just as an FYI: we talked about it in the planning for this sprint and delayed this change until the last two week sprint
-> I'll merge this PR later and this **must** be done in the last two weeks of the project!